aptos-indexer-processor-example/src/utils/starting_version.rs

```diff
@@ -41,6 +44,18 @@ pub async fn get_latest_processed_version_from_db(
 ) -> Result<Option<u64>> {
     let mut conn = conn_pool.get().await?;
 
+    if let Some(backfill_config) = &indexer_processor_config.backfill_config {
+        return match BackfillProcessorStatusQuery::get_by_processor(
+            &backfill_config.backfill_alias,
+            &mut conn,
+        )
+        .await?
+        {
+            Some(status) => Ok(Some(status.last_success_version as u64 + 1)),
```

Comment:
Since we're already here, we should probably move the `+1`  to a higher level, this function seems like the wrong place for it.

Reply:
makes sense lol it disagrees with the name of the function. We now have:

```
get_starting_version(): 
  return get_latest_processed_version_from_db() + 1
```